snakefmt/parser/syntax.py

```diff
@@ -50,6 +55,8 @@ def operator_skip_spacing(prev_token: Token, token: Token) -> bool:
         return True
     elif prev_token.type == tokenize.NAME and token.string == "(":
         return True
+    elif prev_token.type == tokenize.STRING and token.string == ",":
+        return True
```

Comment:
Is this related to #207 and #210?

Reply:
This was actually related to an existing test we have that failed under python3.12 https://github.com/snakemake/snakefmt/blob/a80fa262dfef3b25eed845cd5402cb536162dcab/tests/test_formatter.py#L1053-L1063

It basically changed 
```py
f\"./bin/notify-on-start {{config.get('build_name', 'unknown')}} {{SLACK_TS_FILE}}\"\n"
# to                                              | space added here  
f\"./bin/notify-on-start {{config.get('build_name' , 'unknown')}} {{SLACK_TS_FILE}}\"\n"

Reply:
thanks